TODOS.md

```diff
@@ -0,0 +1,7 @@
+Last updated: Tue Jan 15 2019 - 11:27:58 AM
```

Comment:
The timestamp should be left out as it will otherwise result in a diff every time the script is run regardless of whether comments were changed or not.